packages/design-system/src/elements/toniq-date-time/toniq-date-time.element.ts

```diff
@@ -1,47 +1,45 @@
 import {isTruthy} from '@augment-vir/common';
-import {FullDate, toLocaleString} from 'date-vir';
+import {FullDate} from 'date-vir';
 import {html} from 'element-vir';
+import {HumanizeDurationUnit, formatFullDate, humanizeDuration} from '../../duration';
 import {defineToniqElement} from '../define-toniq-element';
 
-function formatFullDate(fullDate: FullDate): {
-    date: string;
-    time: string;
-} {
-    /**
-     * Note that the below .replace strings are to fix weird behavior in Chromium where the "NARROW
-     * NO-BREAK SPACE" character (code 8239) is used instead of a space (code 32) before AM/PM.
-     */
-    return {
-        date: toLocaleString(fullDate, {dateStyle: 'medium'}).replace(/\s/g, ' '),
-        time: toLocaleString(fullDate, {timeStyle: 'short'}).replace(/\s/g, ' '),
-    };
-}
-
 export const ToniqDateTime = defineToniqElement<{
     fullDate: FullDate;
-    parts: {
+    parts?: {
         date: boolean;
         time: boolean;
     };
+    isHumanize?: boolean;
```

Comment:
We can remove this input and use `inputs.allowedRelativeUnits ?` alone as the check for using the relative string format.